ramls/userSummary.json

```diff
@@ -42,6 +33,10 @@
           "recall": {
             "description": "Indicates if the item was recalled",
             "type": "boolean"
+          },
+          "itemLost" : {
```

Comment:
itemIsLost?

Reply:
Does it add any more clarity to what this flag represents?

Reply:
From my point of view, it will make more clear the type of variable from its name...